index.html

```diff
@@ -71,6 +72,27 @@
             </li>
           </ul>
         </div>
+
+        <div class="tile-btn-container">
```

Comment:
Ik zie deze niet op mobiel

Reply:
Klopt heb m op mobiel op 1 tile en vanaf grotere schermen de optie, zal ik vanaf mobiel al de 1 of 2 tile optie geven?